tensorflow-examples/src/main/java/org/tensorflow/model/examples/objectdetection/FasterRcnnInception.java

```diff
@@ -0,0 +1,302 @@
+package org.tensorflow.model.examples.objectdetection;
+/*
+
+download the model from https://tfhub.dev/tensorflow/faster_rcnn/inception_resnet_v2_1024x1024/1
```

Comment:
This comment should be cleaned up to make it clear how to use the example. Or alternatively put this text in a markdown file and reference the code.

Reply:
Will do.